src/App/App.js

```diff
@@ -31,14 +56,14 @@ class App extends Component {
         <nav>
           <NavBar
           toggled={this.state.toggled}
```

Comment:
Consider doing some of the same destructuring for state, as you have for props elsewhere, eg -

`const { toggled, movies } = this.state`